hindi/Akshay_Kumar/akshay_kumar.md

```diff
@@ -1,4 +1,4 @@
-Here are some facts given about Akshay Kumar, to know more 
-click [here](https://en.wikipedia.org/wiki/Akshay_Kumar)
-
+[Here](https://en.wikipedia.org/wiki/Akshay_Kumar) is a link to his wiki page.<br />
+[Here](https://www.facebook.com/akshaykumarofficial/) is a link to his facebook page.<br />
+Click [here](..) to go back to parent directory.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.